x-pack/test/security_solution_cypress/cypress/tasks/create_new_case.ts

```diff
@@ -65,6 +65,7 @@ export const attachTimeline = (newCase: TestCase) => {
       cy.get('body').type('{esc}');
       cy.get(INSERT_TIMELINE_BTN).click();
       cy.get(LOADING_INDICATOR).should('not.exist');
+      cy.get('[data-test-subj="selectable-input"]').click();
```

Comment:
This test was failing due to tooltip on the search_box.
This click is added to get past tooltip and let search_box be visible:
![image](https://github.com/elastic/kibana/assets/117571355/4882ea05-58ca-4ac5-9c08-337f14352f4d)